src/main/java/org/aesh/console/command/invocation/CommandInvocation.java

```diff
@@ -114,7 +120,16 @@
      *
      * @param input command input
      */
-    void executeCommand(String input) throws InterruptedException;
+    void executeCommand(String input) throws CommandNotFoundException,
```

Comment:
when the command runs in a "live" session/terminal it can be interrupted. hm, it might not be needed to explicitly define that it might throw an interruptedexception, but æsh need to be able to catch it.